spec/system/support_interface/cancel_reference_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+RSpec.feature 'Cancelling references' do
+  include DfESignInHelpers
+
+  scenario 'Support agent cancels reference' do
+    given_i_am_a_support_user
+    and_there_is_an_application
+    and_i_visit_the_application
+
+    when_i_click_to_cancel_the_reference
+    and_i_confirm_i_really_want_to_cancel_the_reference
+    then_the_reference_is_cancelled
```

Comment:
Thought it would have been cool if this test went on to check that the candidate is then able to add a replacement referee, but understand that would be out of the scope of this particular system test which is support-only